tests/cli/main.tf

```diff
@@ -1,5 +1,12 @@
 resource "terraform_data" "opslevel_cli" {
   provisioner "local-exec" {
-    command = "opslevel ${var.command}"
+    command = var.command == null ? ":" : "opslevel ${var.command}"
+  }
+}
+
+resource "terraform_data" "delete_command" {
```

Comment:
😆 - is this your hacky work around to the provisioner doesn't run twice?

Reply:
Yes and go ahead. It's ok to be amazed 🤣